src/ralph/networks/models/networks.py

```diff
@@ -394,6 +393,18 @@ def reserved_top(self):
         # DEPRECATED
         return self.reserved_from_end
 
+    @property
+    def dns_servers_in_order(self):
+        from ralph.dhcp.models import DNSServer
+        servers = None
+        if self.dns_servers_group:
+            servers = DNSServer.objects.filter(
```

Comment:
wouldn't it generate extra query(queries) for every network, when iterate in template? Consider prefetch_related first (here, I guess https://github.com/allegro/ralph/pull/3171/files#diff-a97db8bac29d9b8096414a7656131436L228) , and here iterate over `self.dns_servers_group. server_group_order.all()`